### DISCUSSION TITLE
Agent-based NLP Product -- Looking for help/devs!

### DISCUSSION THREAD

**Discussion by bgoober** [fetchai/uAgents#213] [uAgent Ideas]
Hello all!

I am building an agent-based product on top of fetch.ai's micro-agent framework and I could use some help completing the MVP!

I am using the HuggingFace integrations to perform some NLP tasks and return the information to the user.

I am close to completing the MVP of the product (it currently runs 100% locally), but need some help with front-end/back-end communication via a local Flask server.

The front-end is JavaScript based. The back-end is python, using the established t5-base integration.

I could use some help with:

- [ ] JS front-end
- [ ] Python (agents) back-end
- [ ] Flask server (local, python)

If you are interested please reach out, or contact me via friend request/dm on discord at ben_ptd0

I am also in the fetch.ai discord server, so you can ping me there as well.

---

Beyond the MVP, I am looking to progress to dedicated funding or a development grant in order to turn this into a revenue-generating product. So if you are looking for a potential long-term project with potential for equity/ownership as a developer then please reach out!

**Comment by Wallacerogersjr** [comment]
Hello,

I am working on a similar system on Fetch.ai through a nonprofit. Let’s
talk and maybe collaborate

Let me know
Thanks Wallace Rogers

**Reply by bgoober** [reply]
Hey Wallace, thanks for the reply! Do you have a github repo, or site for the project? I'd love to take a look! It's for a non-profit? Which non-profit is it?

**Reply by Autonomousweb3Dev** [reply]
@bgoober why you closed the discussion?

**Comment by Wallacerogersjr** [comment]
I am sorry, been soo busy, day and night, soon I will try an continue
